pom.xml

```diff
@@ -11,14 +11,14 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>ui-grid</artifactId>
-    <version>3.0.1-rc.11-SNAPSHOT</version>
+    <version>3.0.1-rc.16-SNAPSHOT</version>
```

Comment:
This should be `3.0.0-rc.16-SNAPSHOT` since Maven does the actual release (strips off the `-SNAPSHOT`.